src/releasers/java-yoshi.ts

```diff
@@ -45,6 +45,13 @@ const CHANGELOG_SECTIONS = [
 ];
 
 export class JavaYoshi extends ReleasePR {
+  constructor(options: ReleasePROptions) {
+    if (!options.label && options.snapshot) {
+      options.label = 'type: process';
```

Comment:
this codebase is quickly becoming object oriented :smile: I'm glad I refactored these classes out.